tests/api.c

```diff
@@ -59929,13 +59931,22 @@ static int test_wolfSSL_EC_POINT(void)
     hexStr = EC_POINT_point2hex(group, Gxy, POINT_CONVERSION_UNCOMPRESSED, ctx);
     ExpectNotNull(hexStr);
     ExpectStrEQ(hexStr, uncompG);
+    AssertNotNull(get_point = EC_POINT_hex2point(group, hexStr, NULL, ctx));
```

Comment:
@kojo1  `get_point` is being leaked here. https://cloud.wolfssl-test.com/jenkins/job/PRB-valgrind-check-v3/3027/pipeline-console/

Reply:
```==3601847== 40 bytes in 1 blocks are still reachable in loss record 3 of 28
==3601847==    at 0x483DD99: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==3601847==    by 0x1F3489: wolfSSL_EC_POINT_new (in /var/lib/jenkins/workspace/PRB-valgrind-check-v3@4/wolfssl/tests/unit.test)
==3601847==    by 0x12FB47: test_wolfSSL_EC_POINT (in /var/lib/jenkins/workspace/PRB-valgrind-check-v3@4/wolfssl/tests/unit.test)
==3601847==    by 0x1B5001: ApiTest (in /var/lib/jenkins/workspace/PRB-valgrind-check-v3@4/wolfssl/tests/unit.test)
==3601847==    by 0x11BFA9: unit_test (in /var/lib/jenkins/workspace/PRB-valgrind-check-v3@4/wolfssl/tests/unit.test)
==3601847==    by 0x48B2082: (below main) (libc-start.c:308)```

Reply:
It was a buffer overflow. Needed one more byte.

char   strGx[MAX_ECC_BYTES * 2 + 1];